connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -313,7 +289,7 @@ export async function syncZendeskCategoryActivity({
  * This activity is responsible for syncing the next batch of articles to process.
  * It does not sync the Category, only the Articles.
  */
-export async function syncZendeskArticlesBatchActivity({
+export async function syncZendeskArticleBatchActivity({
```

Comment:
I thought we said we'd leave the "s" ?
Seems like this is refactored on every PR I'm reviewing 😂

Reply:
On second thought I'm reverting this to the original implementation: what these 2 activities do is to sync a batch of tickets or articles, there is no such thing as a "batch activity" (if there were the name would be misleading anyway) and _this naming is more consistent with what is done elsewhere_ (haven't checked but betting on it).

Reply:
This is not refactored on every PR you review, you reviewed the same PR twice (this one is a subset of 8567)

Reply:
GJ, let's keep without s ➡️ https://dust.tt/w/0ec9852c2f/assistant/UkbY1FpxT6